src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.cpp

```diff
@@ -237,6 +237,27 @@ FishboneMoncriefDisk::variables(
   return {std::move(specific_internal_energy)};
 }
 
+template <typename DataType, bool NeedSpacetime>
+tuples::TaggedTuple<hydro::Tags::Temperature<DataType>>
+FishboneMoncriefDisk::variables(
```

Comment:
From CI it looks like this needs an explicit instantiation at the end of this file.